src/store/modules/chats.js

```diff
@@ -29,9 +37,28 @@ export default {
       let numUnread = ids.length - lastUnreadIndex - 1
       return numUnread
     },
+    getUnreadValue: (state) => (addr) => {
+      return calculateUnreadValue(state, addr)
+    },
     getLastRead: (state) => (addr) => {
       return state.data[addr].lastRead
     },
+    getValuedChatOrder (state) {
+      return state.order.map(
+        addr => ({
+          address: addr,
+          unreadValue: calculateUnreadValue(state, addr)
+        })
+      ).sort(({ address: addressA, unreadValue: valueA }, { address: addressB, unreadValue: valueB }) => {
```

Comment:
This is kinda annoying.  But it works for now.  We should probably have some normal order override for things that have no unread messages.